Makefile

```diff
@@ -222,11 +222,28 @@ generate:
 	controller-gen object:headerFile="hack/boilerplate.go.txt" paths="$(GO_PACKAGES)"
 .PHONY: generate
 
+HELM_CHANNEL ?=latest
+HELM_CHARTS=scylla-operator scylla-manager scylla
```

Comment:
nit: conventionally global makefile variables are kept together before defining target so people can easily see what is available when opening the makefile